pkgs/by-name/li/libguestfs-appliance-debian/deb-closure-amd64.nix

```diff
@@ -0,0 +1,2548 @@
+# This is a generated file.  Do not modify!
+# Following are the Debian packages constituting the closure of: base-passwd dpkg libc6-dev perl bash dash gzip bzip2 tar grep mawk sed findutils g++ make curl patch locales coreutils util-linux file dpkg-dev pkg-config login passwd sysvinit diff libguestfs-tools linux-image-amd64
+
+{ fetchurl }:
+
+[
+
+  [
+
+    (fetchurl {
+      url = "https://snapshot.debian.org/archive/debian/20250227T210548Z/pool/main/g/gcc-14/gcc-14-base_14.2.0-16_amd64.deb";
+      sha256 = "7c2063ec46eaf8f161a5643712aeb114905e20dc02236531455c0b1810aabadb";
+    })
+
+  ]
+
+  [
+
+    (fetchurl {
+      url = "https://snapshot.debian.org/archive/debian/20250227T210548Z/pool/main/g/gcc-14/libgcc-s1_14.2.0-16_amd64.deb";
+      sha256 = "ddd3be2987cf01718d29aa0832ab7799cb818a54148938d4289ca27aa3811c86";
+    })
+
+    (fetchurl {
+      url = "https://snapshot.debian.org/archive/debian/20250227T210548Z/pool/main/g/glibc/libc6_2.40-7_amd64.deb";
+      sha256 = "5072445cef9e11283bd5aa9ceb3debdc737baf0c93ae77949096ae6fc47a2c96";
+    })
+
+  ]
```

Comment:
The generator should not put so much whitespace in this file.

Reply:
This comes from a perl script in vmTools. I don't know any perl, should I try to fix it anyway?

Reply:
Managed to fix it.